lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -67,7 +67,17 @@ EOF
   echo "Install Magma"
   apt-get update -y
   apt-get upgrade -y
-  apt-get install curl zip python3-pip docker.io net-tools sudo docker-compose -y
+  apt-get install curl zip python3-pip net-tools sudo ca-certificates gnupg lsb-release -y
+  
+  mkdir -p /etc/apt/keyrings
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+  echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
```

Comment:
Could we use the long option `--codename --short` for better readability?

Reply:
What about a general readability cleanup regarding these flags? The specific call appears 12 times in the magma repo. A separate cleanup issue would be better suited for this in my opinion.

Reply:
Could you create an issue please? This could also be a `goodfirstissue`.

Reply:
https://github.com/magma/magma/issues/14830